R/predict_race.R

```diff
@@ -191,7 +191,7 @@ predict_race <- function(voter.file, census.surname = TRUE, surname.only = FALSE
   
   if((model == "BISG")){
     cl <- c(as.name("predict_race_new"), cl)
-    preds <- eval.parent(as.call(cl))
+    preds <- eval(as.call(cl))
```

Comment:
It has to be `eval.parent` (the same as, `eval(..., env = parent.frame())` because of scoping. Otherwise all the tests will fail because the arguments will not evaluate from `match.call()` and data objects will not be passed.